src/cloud-functions.ts

```diff
@@ -466,23 +467,20 @@ export function makeCloudFunction<EventData>({
         };
       }
 
-      if (Object.keys(labels).length > 0) {
-        endpoint.labels = { ...endpoint.labels, ...labels };
-      }
+      // Note: We intentionally don't make use of labels args here.
+      // labels is used to pass SDK-defined labels to the trigger, which isn't
+      // something we will do in the container contract world.
+      endpoint.labels = { ...endpoint.labels };
 
       return endpoint;
     },
   });
 
   if (options.schedule) {
     cloudFunction.__requiredAPIs = [
-      {
-        api: 'pubsub.googleapis.com',
-        reason: 'Needed for v1 scheduled functions.',
-      },
       {
         api: 'cloudscheduler.googleapis.com',
-        reason: 'Needed for v1 scheduled functions.',
```

Comment:
here's a case where I could argue for using "v1" because v2 will not use Pub/Sub.

Alternatively you can just remove this required API because it's part of the APIs that get enabled with Cloud Functions.

Reply:
I don't think Cloud Scheduler gets enabled as part of GCF, but I'm not sure what enables it since I do see it enabled when creating a new GCP project 🤔 . I'm going to add 'v1' to err on side of being safe, but can remove later when I dig in and figure out if we can rely on the cloud scheduler API being enabled on users project by some means.